intera_interface/src/intera_interface/gripper_factory.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2013-2017, Rethink Robotics Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import rospy
+from intera_core_msgs.msg import (
+    IONodeConfiguration,
+    IONodeStatus,
+    IOComponentConfiguration
+)
+import intera_dataflow
+from intera_interface import (
+    Gripper,
+    SimpleClickSmartGripper
+)
+
+
```

Comment:
What are your thoughts on an external helper function?
```
def get_current_gripper_interface():
  gf = GripperFactory()
  return gf.get_current_gripper_interface()
```

Reply:
The external helper function is starting to feel better.

So you thinking the usage would be something like:
```
gripper = intera_interface.get_current_gripper_interface()
```

?

Reply:
Yes, that's the thought. What do you think from a usability perspective?